api.md

```diff
@@ -469,7 +528,81 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
+
+Detach a data node from one hypertable or from all hypertables.
+
+Reasons for detaching a data node:
 
+- A data node should no longer be used by a hypertable and needs to be
+removed from all hypertables that use it
+- You want to have fewer data nodes for a distributed hypertable to
+partition across to other distributed hypertables in the database
+
+>:WARNING: This function *MUST* be run on an access node.
+
+#### Required Arguments [](detach_data_node-required-arguments)
+
+| Name        | Description                       |
+|-------------|-----------------------------------|
+| `node_name` | Local name of data node to detach |
```

Comment:
I think my problem here is that there isn't really such thing as a "remote data node name".  There the name of the database hosting the data node on the remote instance, but that seems a different concept to me.

Reply:
I removed all mentions of "Local name"